src/components/join-flow/join-flow-steps.scss

```diff
@@ -22,6 +22,23 @@
     transform: translate(21.5625rem, 0);
 }
 
+.validation-birthdate-input {
+    transform: translate(8.75rem, .25rem);
```

Comment:
What are these numbers based on? Is it the length of some text? If so, this won't be right in other languages.

Reply:
That's a great question! It's based on the fixed width of the select form elements, which is actually set below in `select.join-flow-select`. So it should not be language-dependent.